### DISCUSSION TITLE
Downtime for https://searxng.tordenskjold.de

### DISCUSSION THREAD

**Discussion by CaptainTordenskjold** [searxng/searx-instances#302] [Report instance downtime]
My instance may be unavailable several times next week. I will do some tests with other services, and meanwhile I can't guarantee that the instance will be reachable all the time.
I guess it will take a few days. I only have time to work on it after my work hours.
I would suggest to temporarily remove my instance from searx.space, and when I'm done I'll let you know so it can be added back.

**Comment by unixfox** [comment]
Thank you for the notification.
I temporarily removed your instance here: https://github.com/searxng/searx-instances/commit/3a4a1e60feef11502bfb5ee6a7db70dae41630eb. Please ask to add it back when you have finished your tests.

**Comment by CaptainTordenskjold** [comment]
Hi @unixfox, i just can't spend the time I want. You can delete my instance completely. Should I open an extra issue for that? Or is this enough?

**Reply by unixfox** [reply]
Sad to see your instance shutting down. You do not have to do anything else.
